js/glossary.js

```diff
@@ -127,13 +128,13 @@ Glossary.prototype.show = function(e) {
 Glossary.prototype.hide = function() {
   this.$body.removeClass('is-open').attr('aria-hidden', 'true');
   this.$toggle.removeClass('active');
-  this.$trigger.focus();
+  this.$trigger && this.$trigger.focus();
```

Comment:
Right. I actually got rid of that warning on the webapp--I can copy over our linter config here to avoid the complaint.